app/models/waste_carriers_engine/transient_registration.rb

```diff
@@ -102,7 +102,7 @@ def renewal_application_submitted?
     end
 
     def pending_payment?
```

Comment:
I was expecting to see the spec for this method to change as well with an `expect(subject).to receive(:unpaid_balance?) ....`, shall we update it?

Reply:
Fixed: https://github.com/DEFRA/waste-carriers-engine/pull/534/commits/b4d267b38f72e90177cfd7b70fb39df567b932d7